main.py

```diff
@@ -1,22 +1,30 @@
-import argparse
 import os
+import time
+import argparse
 
 import uvicorn
 from fastapi import FastAPI, UploadFile
 from fastapi.encoders import jsonable_encoder
+from fastapi.responses import PlainTextResponse
+from starlette.middleware.base import BaseHTTPMiddleware
+os.environ['PROMETHEUS_DISABLE_CREATED_SERIES'] = 'true'
+# pylint: disable=wrong-import-position
+from prometheus_client import generate_latest, Counter, Histogram, Gauge, REGISTRY
```

Comment:
A better design is to put related functions into another file and just call them from main.